lib/ansible/modules/cloud/azure/azure_rm_virtualmachine.py

```diff
@@ -776,6 +795,25 @@ def extract_names_from_blob_uri(blob_uri, storage_suffix):
     return extracted_names
 
 
+def format_blob_uri(account, storage_suffix, blob, container='vhds'):
+    container = container or 'vhds'  # avoid None
+    return 'https://{0}.blob.{1}/{2}/{3}'.format(account, storage_suffix, container, blob)
+
+
+data_disk_spec = dict(
+    lun=dict(type='int', required=True),
+    disk_size_gb=dict(type='int'),
+    managed_disk_type=dict(type='str', choices=['Standard_LRS', 'StandardSSD_LRS', 'UltraSSD_LRS', 'Premium_LRS']),
+    managed_disk_id=dict(type='str'),
+    storage_account_name=dict(type='str'),
+    storage_container_name=dict(type='str'),
+    storage_blob_name=dict(type='str'),
+    caching=dict(type='str', choices=['empty', 'Empty', 'read_only', 'ReadOnly', 'read_write', 'ReadWrite']),
```

Comment:
same, probably stick with the CamelCase form already in use through the rest of the module